pkg/proxy/winkernel/hnsV1.go

```diff
@@ -137,6 +147,10 @@ func (hns hnsV1) createEndpoint(ep *endpointsInfo, networkName string) (*endpoin
 		hnsID:           createdEndpoint.Id,
 		providerAddress: ep.providerAddress, //TODO get from createdEndpoint
 		hns:             hns,
+
```

Comment:
The caller of CreateEndpoint in proxier.go for winkernel is not populating the ready/serving/terminating state, so these would be nil